src/main/java/com/minecolonies/core/colony/permissions/ColonyPermissionEventHandler.java

```diff
@@ -173,12 +184,23 @@ private void cancelEvent(final Event event, @Nullable final Entity entity, final
 
             final long worldTime = entity.level.getGameTime();
             if (!lastPlayerNotificationTick.containsKey(entity.getUUID())
-                  || lastPlayerNotificationTick.get(entity.getUUID()) + (Constants.TICKS_SECOND * 10)
+                  || lastPlayerNotificationTick.get(entity.getUUID()) + (TICKS_SECOND * 10)
                        < worldTime)
             {
                 MessageUtils.format(PERMISSION_DENIED).sendTo((Player) entity);
                 lastPlayerNotificationTick.put(entity.getUUID(), worldTime);
             }
+            else
+            {
+                if (playerAttempts.compute(entity.getUUID(), (uuid, count) -> count == null ? 1 : count + 1) > 10)
```

Comment:
Does this get reset after the notification tick?
Right now, it looks like if you try to do something that is not allowed a couple of times, then wait a minute, then a couple of times again, then wait a minute and continue,
eventually you seem to trigger the levitation, without ever breaking the permission more than ten times within ten seconds. Is it intended to work like that?

Reply:
good point, changed